tanjun/abc.py

```diff
@@ -462,6 +595,30 @@ async def respond(
             typing.Union[hikari.SnowflakeishSequence[hikari.PartialRole], bool]
         ] = hikari.UNDEFINED,
     ) -> typing.Optional[hikari.Message]:
+        """Respond with the content to this context.
+
+        Parameters
+        ----------
+        content : hikari.UndefinedOr[typing.Any]
+            The content to respond with.
+
+        Other Parameters
+        ----------------
+        ensure_result : bool
+            If provided and set to `True`, The response will be fetched performing an API call.
```

Comment:
This doesn't always result in it performing an API call, would be better to say that it may perform an extra API call if necessary to ensure that it has a message object to return